### ISSUE TITLE
 Dice loss probelm

### ISSUE THREAD

**Issue by Defneh** [AshishSingh2261/Ingenium_NeurAi#1]
When I use categorical cross-entropy, like in your code from brain_tumor_segmentation_u_net.ipynb, the IOU and Dice coefficients work fine. However, when I switch to Dice loss or other loss functions, the loss doesn't change.

Could you please help me figure out why?

When I print the values, I get the following:

y_batch: min/max: 0.0 / 1.0
Model output (y_pred) min/max: 0.08728714 / 0.43650725

Could this discrepancy be the cause of the issue?
Thank you
